File: example.BUILD.bazel.go
load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_binary")

go_library(
    name = "go_default_library",
    srcs = ["hello.go"],
    importpath = "example.com/monorepo/go/hello",
    visibility = ["//visibility:public"],
)

go_binary(
    name = "hello_bin",
    srcs = ["main.go"],
    deps = [":go_default_library"],
)
# This BUILD.bazel file defines a Go library and a Go binary target.

File: example.BUILD.bazel.java
load("@bazel_tools//tools/jdk:default_java_toolchain.bzl", "default_java_toolchain")
# Use the `java_library` rule (no external-dep example here)
java_library(
    name = "mylib",
    srcs = glob(["src/main/java/**/*.java"]),
    visibility = ["//visibility:public"],
)
